testing/database.rst

```diff
@@ -86,7 +86,7 @@ it's easy to pass a mock object within a test::
 
             // Now, mock the repository so it returns the mock of the employee
             $employeeRepository = $this
-                ->getMockBuilder(EntityManagerInterface::class)
+                ->getMockBuilder(EntityRepository::class)
```

Comment:
I always prefer to makes those kind of changes simultaneously for all existing articles, to avoid differences between articles. Maybe we can open an issue to not forget about that. Thanks!